modules/rup/internacion/censo.routes.ts

```diff
@@ -33,4 +34,130 @@ router.get('/censo-mensual', Auth.authenticate(), asyncHandler(async (req: any,
+                        } else {
+                            censoDiario.push(movimiento);
+                        }
+                    } else {
+                        censoDiario.push(movimiento);
+                    }
+                }
+            }
+        });
+    });
+
+    res.set('Content-Type', 'text/csv');
+    res.setHeader('Content-disposition', 'attachment; filename=censo-diario.csv');
+    csv.write(censoDiario, {
+        headers: true, transform: (row) => {
+            return {
+                Paciente: `${row.datos.paciente.apellido}, ${row.datos.paciente.nombre} | ${row.datos.paciente.documento}`,
+                Cama: `${row.datos.cama.nombre}, ${row.datos.cama.sectores[row.datos.cama.sectores.length - 1].nombre}` || `${row.datos.cama.nombre}`,
+                Ingreso: row.ingreso || '',
+                'Pase de': row.paseDe || '',
+                Egreso: row.egreso || '',
+                'Pase a': row.paseA || '',
+                'Fecha de ingreso': row.fechaIngreso ? moment(row.fechaIngreso).format('DD/MM/YYYY hh:mm') : '',
+                'Dias de estada': row.diasEstada || ''
+            };
+        }
+    }).pipe(res);
+}));
+
+router.post('/censo-mensual/csv', Auth.authenticate(), asyncHandler(async (req: any, res) => {
+    const organizacion = Auth.getOrganization(req);
+    const result = await CensosController.censoMensual({
+        organizacion,
+        unidadOrganizativa: req.body.unidadOrganizativa,
+        fechaDesde: req.body.fechaDesde,
+        fechaHasta: req.body.fechaHasta
+    });
+    const totales = {
+        censo: {
+            existenciaALas0: 0,
+            ingresos: 0,
+            pasesDe: 0,
+            altas: 0,
+            defunciones: 0,
+            pasesA: 0,
+            existenciaALas24: 0,
+            ingresosYEgresos: 0,
+            pacientesDia: 0,
+            diasEstada: 0,
+            disponibles: 0
+        }
+    };
+
+    result.map(r => {
+        totales.censo.existenciaALas0 += r.censo.existenciaALas0;
+        totales.censo.ingresos += r.censo.ingresos;
+        totales.censo.pasesDe += r.censo.pasesDe;
+        totales.censo.altas += r.censo.altas;
+        totales.censo.defunciones += r.censo.defunciones;
+        totales.censo.pasesA += r.censo.pasesA;
+        totales.censo.existenciaALas24 += r.censo.existenciaALas24;
+        totales.censo.ingresosYEgresos += r.censo.ingresosYEgresos;
+        totales.censo.pacientesDia += r.censo.pacientesDia;
+        totales.censo.diasEstada += r.censo.diasEstada;
+        totales.censo.disponibles += r.censo.disponibles;
+    });
+    result.push(totales);
+
+    res.set('Content-Type', 'text/csv');
+    res.setHeader('Content-disposition', 'attachment; filename=censo-mensual.csv');
+    csv.write(result, {
+        headers: true, transform: (row) => {
+            return {
+                Fecha: row.fecha ? moment(row.fecha).format('DD/MM/YYYY hh:mm') : 'TOTALES',
```

Comment:
No tiene sentido la hora, la fecha va en formato ingles así se puede ordenar en el csv. 
```suggestion
                Fecha: row.fecha ? moment(row.fecha).format('YYYY-MM-DD') : 'TOTALES',
```